src/pages/resultsView/mutation/DriverAnnotationProteinImpactTypeBadgeSelector.tsx

```diff
@@ -220,9 +235,17 @@ export default class DriverAnnotationProteinImpactTypeBadgeSelector extends Prot
             this.props.onSelect(selectedOption, allValuesSelected);
     }
 
+    @action.bound
+    protected onSettingMenuClick(e: React.MouseEvent<any>) {
+        e.stopPropagation(); // Prevent click being applied to parent element
+        this.settingMenuVisible = !this.settingMenuVisible;
+        this.props.onClickSettingMenu &&
+            this.props.onClickSettingMenu(this.settingMenuVisible);
+    }
+
     public render() {
         return (
-            <div>
+            <div className="legend-panel">
```

Comment:
this is annoying but we should use css modules approach here b/c legend-panel is too general.

Reply:
that will involve converting mutations.scss to mutations.module.scss below.